src/Symfony/Component/PropertyInfo/Tests/Extractor/PhpStanExtractorTest.php

```diff
@@ -968,7 +1042,18 @@ public static function allowPrivateAccessProvider(): array
 
     public function testGenericInterface()
     {
-        $this->assertNull($this->extractor->getTypes(Dummy::class, 'genericInterface'));
+        $this->assertEquals(
+            [
+                new LegacyType(
+                    builtinType: LegacyType::BUILTIN_TYPE_OBJECT,
+                    class: BackedEnum::class,
```

Comment:
```suggestion
                    class: \BackedEnum::class,
```

Reply:
Shouldn't automated cs check in CI check that?

Reply:
@nicolas-grekas is fabbot using an uptodate version of PHP-CS-Fixer ? The `global_namespace_import` fixer should be enforcing that already (and it is part of the Symfony ruleset)